src/applications/appeals/10182/content/contestableIssues.jsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import { format } from 'date-fns';
+import AdditionalInfo from '@department-of-veterans-affairs/component-library/AdditionalInfo';
+
+// import { scrollTo } from '../helpers';
+
+// We shouldn't ever see the couldn't find contestable issues message since we
+// prevent the user from navigating past the intro page; but it's here just in
+// case we end up filtering out deferred and expired issues
+export const EligibleIssuesTitle = props =>
+  props?.formData?.contestedIssues?.length === 0 ? (
+    <h2 className="vads-u-font-size--h4" name="eligibleScrollElement">
+      Sorry, we couldn’t find any eligible conditions
+    </h2>
+  ) : (
+    <legend name="eligibleScrollElement" className="vads-u-font-size--lg">
+      Please select the condition(s) you would like to have reviewed
+      <span className="schemaform-required-span vads-u-font-weight--normal vads-u-font-size--base">
+        (*Required)
+      </span>
+    </legend>
+  );
+
+export const EligibleIssuesDescription = (
+  <>
+    <div>
+      <p className="vads-u-margin-top--0">
+        These issues are in your VA record. If an issue is missing from this
+        list, you can add it by clicking the <strong>Add condition</strong>{' '}
+        button (NOT YET IMPLEMENTED).
```

Comment:
Is the NOT YET IMPLEMENTED intentional?

Reply:
Yes it is.... we have to present this to DEPO on Friday and the design is not finalized.